README.md

```diff
@@ -92,12 +108,17 @@ Here's a hands-on exercise to illustrate the power of GitOps:
 The beauty of GitOps lies in its ability to seamlessly reflect your git repository's state onto your cluster. Should you decide that the golden glimmer of podinfo doesn't quite suit your taste, reverting is a breeze:
 
 1. Whip open your terminal and run `git log` to retrieve the hash of the commit you wish to undo.
+
+> Connect this with the Gitlens extension, e.g. link directly to Git history? Maybe too fancy for now.
```

Comment:
mhh I'd like to not add that complexity. especially if users didn't use gitlens before it may even be confusing and add cognitive load (it does for me). I think for this single command git log does its job well. However, I could add the git revert HEAD~n to allow the user to undo n commits. Happens sometimes that users did 2 commits e.g. they made an error first or changed color 2 times.

Reply:
Yeah, probably overengineered